src/hooks/useFilterBlockedMessageIds.ts

```diff
@@ -10,17 +12,28 @@ export default function useFilterBlockedMessageIds(
   const { data: blockedInHub } = getBlockedResourcesQuery.useQuery({
     spaceId: hubId,
   })
-  const { data: blockedInChat } = getBlockedResourcesQuery.useQuery({
-    postId: chatId,
-  })
+  const { data: chat } = getPostQuery.useQuery(chatId)
+  const { data: blockedInChat } = getBlockedResourcesQuery.useQuery(
+    {
+      postEntityId: chat?.entityId ?? '',
+    },
+    { enabled: !!chat?.entityId }
+  )
+  const appId = getAppId()
+  const { data: blockedInApp } = getBlockedResourcesQuery.useQuery(
+    { appId },
+    { enabled: !!appId }
```

Comment:
Looks like we can do additional wrapper, because you do the 

  const appId = getAppId()

  { enabled: !!appId }


  In many files